iep-ses-monitor/src/main/scala/com/netflix/iep/ses/SesMonitoringService.scala

```diff
@@ -127,50 +127,74 @@ class SesMonitoringService @Inject()(
     MessageAction.delete(message)
   }
 
+  private def getPathAsString(obj: Map[String, Any], path: String*) = {
```

Comment:
```suggestion
  private def getPathAsString(obj: Map[String, Any], path: String*): String = {
```

It is generally better to be explicit with return types on the method. Otherwise it can change in strange ways due to inference. Being explicit provides a sanity check from the compiler.

Reply:
Happy to add it. Most style guides I've worked with favor inference on `private` methods and explicit on public, unless it's ambiguous by nature.

I see now that our projects do consistently specify a type, regardless of visibility.